content/S2016/S16-event-code-party.md

```diff
@@ -0,0 +1,22 @@
+title: Code Party 
+Date: 2016-07-14
+Category: Events
+Tags: code party, social, open source, academic 
+Slug: code-party-S16
+Author: Arshia Mufti
+Summary: A casual night of fun and work!
+
+WiCS is back with another end-of-term code party! Work on and share your projects
+with other women in CS. You can also work on homework or assignments with your friends or simply
+drop by, hang out, and mingle!
+
+We'll be bringing snacks and some ideas of things you can work on&mdash;all
```

Comment:
though honestly I think it'd be better as two sentences like Anna's commit - either's fine though *shrug*